### PR TITLE
[ENG-29890] feat: accounting endpoint

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

* Added fields for Accounting GraphQL API
* Added mentions of the new endpoint in other GQL docs
* Added menu link to Accounting GraphQL API Fields

[ENG-29890]: https://aziontech.atlassian.net/browse/ENG-29890?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/devtools/api-graphql/queries/queries.mdx [hunk 1]
```diff
@@ -216,6 +216,118 @@ With the *aggregated model*, you'll receive data according to a time range defin
...
+        "metricSlug": "compute_time",
+        "periodFrom": "2022-06-01",
+        "periodTo": "2022-06-30"
+      }
+    ]
+  }
+}
+```
+
+To query raw data, it’s mandatory to inform:
```
Comment (hannah-sk) [inline src/content/docs/en/pages/devtools/api-graphql/queries/queries.mdx:325]:
```suggestion
To query financial data, it’s mandatory to inform:
```

File: src/content/docs/pt-br/pages/devtools/api-graphql/queries/queries-gql.mdx [hunk 2]
```diff
@@ -215,6 +215,118 @@ Com o *modelo de dados agregados*, você recebe dados de acordo com um intervalo
...
+        "metricSlug": "compute_time",
+        "periodFrom": "2022-06-01",
+        "periodTo": "2022-06-30"
+      }
+    ]
+  }
+}
+```
+
+Para consultar os dados brutos, é obrigatório informar:
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/devtools/api-graphql/queries/queries-gql.mdx:324]:
```suggestion
Para consultar os dados financeiros, é obrigatório informar:
```

File: src/content/docs/en/pages/devtools/api-graphql/overview/overview.mdx [hunk 3]
```diff
@@ -20,6 +20,10 @@ You can use the two Azion GraphQL APIs to perform a logpull of your data and max
 
 <Button href="/en/documentation/devtools/graphql-api/queries/" text="more on aggregated and raw data"></Button>
 
+And you can also use the financial GraphQL API:
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/api-graphql/overview/overview.mdx:23]:
```suggestion
You can also use the financial GraphQL API:
```

File: src/content/docs/pt-br/pages/devtools/api-graphql/queries/queries-gql.mdx [hunk 4]
```diff
@@ -215,6 +215,118 @@ Com o *modelo de dados agregados*, você recebe dados de acordo com um intervalo
...
+    accounted
+    clientId
+    metricSlug
+    periodFrom
+    periodTo
+  }
+}
+```
+
+E a resposta a query:
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/api-graphql/queries/queries-gql.mdx:243]:
```suggestion
E a resposta da query:
```

Comment (MarianaAguilera) [reply src/content/docs/pt-br/pages/devtools/api-graphql/queries/queries-gql.mdx:243]: E a resposta á query?

File: src/content/docs/en/pages/devtools/api-graphql/features/accounting-fields.mdx [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+| Field | Description |
+| ----- | ----------- |
+| accounted | Total amount of use of products per region accounted for the bill detail of the client. Example: `1767504904` |
+| clientId | Client unique identifier on Azion. Example: `8437r` |
+| metricSlug | Identifier of the metric being account. See the complete list of product metrics in the [Terms of Service](/en/documentation/agreements/tos/#3-product-use-calculation-metrics-and-methods) document. Example: `load_balancer_data_transferred` |
+| periodFrom | Start date of the accounted data in year-month-day format. Example: `2022-06-01` |
+| periodTo | End date of the accounted data in year-month-day format. Example: `2022-06-30` |
+| productSlug | Identifier of the product being accounted. Example: `edge_functions` |
+| regionName | Region that the metric was accounted to. Find more information on the [Pricing](/en/documentation/products/pricing/) page. Can be: `United States`, `Canada`, `Europe`, `Brazil`, or `Latam` |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/api-graphql/features/accounting-fields.mdx:26]: Other Regions não aplica aqui?

File: src/content/docs/pt-br/pages/devtools/api-graphql/recursos/accounting-campos.mdx [hunk 6]
```diff
@@ -0,0 +1,26 @@
+---
+title: Campos da API GraphQL de Accounting
+description: Veja as descrições para todos os campos de cada conjunto de dados da API GraphQL de Accounting.
+meta_tags: graphql, api, azion, fields, accounting
+namespace: documentation_graphql_accounting_fields
+permalink: /documentacao/devtools/graphql-api/recursos/campos-gql-accounting/
+menu_namespace: graphqlMenu
+---
+
+Cada [conjunto de dados a GraphQL API](/pt-br/documentacao/devtools/graphql-api/recursos/#conjuntos-de-dados) possui campos específicos que podem ser utilizados ao consultar dados daquele conjunto de dados específico.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/api-graphql/recursos/accounting-campos.mdx:10]:
```suggestion
Cada [conjunto de dados da GraphQL API](/pt-br/documentacao/devtools/graphql-api/recursos/#conjuntos-de-dados) possui campos específicos que podem ser utilizados ao consultar dados daquele conjunto de dados específico.
```

File: src/content/docs/pt-br/pages/devtools/api-graphql/recursos/recursos-gql.mdx [hunk 7]
```diff
@@ -60,7 +60,15 @@ Quatro conjuntos de dados foram descontinuados e substituiídos por outros:
...
 | telemetrySensorsEvents | Eventos relacionados aos sensores dos dispositivos registrados pelo **Azion Mobile SDK**, como toque de tela e informações de giroscópio. |
 
-<Button href="/pt-br/documentacao/devtools/graphql-api/recursos/campos-gql-real-time-events/" text="veja as descrições dos Campos da  API GraphQL do Real⁠-⁠Time Events" variant="secondary"></Button>
+<Button href="/pt-br/documentacao/devtools/graphql-api/recursos/campos-gql-real-time-events/" text="veja as descrições dos Campos da API GraphQL do Real⁠-⁠Time Events" variant="secondary"></Button>
+
+### Accounting GraphQL API
+
+| Conjunto de dados | Descrição |
+| ----------------- | --------- |
+| accountingDetail | Registros relacionados aos dados contabilizados para produtos e suas métricas. |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/api-graphql/recursos/recursos-gql.mdx:69]:
```suggestion
| accountingDetail | Registros relacionados aos dados contabilizados para produtos e suas métricas |
```

File: src/content/docs/en/pages/devtools/api-graphql/features/features.mdx [hunk 8]
```diff
@@ -61,6 +61,14 @@ Four datasets are deprecated and were replaced by other datsets:
 
 <Button href="/en/documentation/devtools/graphql-api/features/gql-real-time-events-fields/" text="see Real-Time Events GraphQL API fields descriptions" variant="secondary"></Button>
 
+### Accounting GraphQL API
+
+| Dataset | Description |
+| ------- | ----------- |
+| accountingDetail | Logs regarding accounted data for products and its metrics. |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/api-graphql/features/features.mdx:68]:
```suggestion
| accountingDetail | Logs regarding accounted data for products and its metrics |
```

File: src/content/docs/pt-br/pages/devtools/api-graphql/visao-geral/visao-geral-gql.mdx [hunk 9]
```diff
@@ -20,6 +20,10 @@ Você pode utilizar as duas APIs GraphQL da Azion para fazer um logpull de seus
 
 <Button href="/pt-br/documentacao/devtools/graphql-api/queries/" text="saiba mais sobre dados agregados e brutos"></Button>
 
+E você também pode usar a API GraphQL de Accounting:
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/api-graphql/visao-geral/visao-geral-gql.mdx:23]:
```suggestion
Você também pode usar a API GraphQL de Accounting:
```

File: src/content/docs/en/pages/devtools/api-graphql/overview/overview.mdx [hunk 10]
```diff
@@ -20,6 +20,10 @@ You can use the two Azion GraphQL APIs to perform a logpull of your data and max
...
+
+- **Accounting GraphQL API**, which provides data related to products and its metrics being accounted in the client's account.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/api-graphql/overview/overview.mdx:25]: não tem link para redirecionar?

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/en/pages/devtools/api-graphql/features/accounting-fields.mdx [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+
+See each available field and their descriptions below.
+
+---
+
+## accountingDetail (Accounting)
+
+| Field | Description |
+| ----- | ----------- |
+| accounted | Total amount of use of products per region accounted for the bill detail of the client. Example: `1767504904` |
```
Comment (helio-neto) [inline src/content/docs/en/pages/devtools/api-graphql/features/accounting-fields.mdx:20]: I don't know if it's a good suggestion, i'm actually an engineer. 🧑‍🏭  😄 
```suggestion
| accounted | Total amount of product usage per region accounted in the bill detail of the client. Example: `1767504904` |
```

**Review by helio-neto — APPROVED** [review]

File: src/content/docs/en/pages/devtools/api-graphql/features/accounting-fields.mdx [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+
+| Field | Description |
+| ----- | ----------- |
+| accounted | Total amount of product usage per region accounted in the bill detail of the client. Example: `1767504904` |
+| clientId | Client unique identifier on Azion. Example: `8437r` |
+| metricSlug | Identifier of the metric being account. See the complete list of product metrics in the [Terms of Service](/en/documentation/agreements/tos/#3-product-use-calculation-metrics-and-methods) document. Example: `load_balancer_data_transferred` |
+| periodFrom | Start date of the accounted data in year-month-day format. Example: `2022-06-01` |
+| periodTo | End date of the accounted data in year-month-day format. Example: `2022-06-30` |
+| productSlug | Identifier of the product being accounted. Example: `edge_functions` |
+| regionName | Region that the metric was accounted to. Find more information on the [Pricing](/en/documentation/products/pricing/) page. Can be: `United States`, `Canada`, `Europe`, `Brazil`, or `Latam` |
```
Comment (hannah-sk) [inline src/content/docs/en/pages/devtools/api-graphql/features/accounting-fields.mdx:26]:
```suggestion
| regionName | Region that the metric was accounted to. Find more information on the [Pricing](/en/documentation/products/pricing/) page. Can be: `United States`, `Canada`, `Europe`, `Brazil`, `Latam`, `All Other Regions`, or `No Region` |
```

File: src/content/docs/pt-br/pages/devtools/api-graphql/recursos/accounting-campos.mdx [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+
+| Campo | Descrição |
+| ----- | --------- |
+| accounted | Quantidade total de uso de produtos por região contabilizada no detalhe da fatura do cliente. Exemplo: `1767504904` |
+| clientId | Identificador único do cliente na Azion. Exemplo: `8437r` |
+| metricSlug | Identificador da métrica sendo contabilizada. Consulte a lista completa de métricas de produtos no documento [Termos de Serviço](/pt-br/documentacao/contratos/tds/#3-metricas-e-metodologias-de-contabilizacao-de-uso-dos-produtos). Exemplo: `load_balancer_data_transferred` |
+| periodFrom | Data de início dos dados contabilizados no formato ano-mês-dia. Exemplo: `2022-06-01` |
+| periodTo | Data de término dos dados contabilizados no formato ano-mês-dia. Exemplo: `2022-06-30` |
+| productSlug | Identificador do produto sendo contabilizado. Exemplo: `edge_functions` |
+| regionName | Região em que a métrica foi contabilizada. Encontre mais informações na página de [Preços](/pt-br/documentacao/produtos/precos/). Pode ser: `United States`, `Canada`, `Europe`, `Brazil`, or `Latam` |
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/devtools/api-graphql/recursos/accounting-campos.mdx:26]:
```suggestion
| regionName | Região em que a métrica foi contabilizada. Encontre mais informações na página de [Preços](/pt-br/documentacao/produtos/precos/). Pode ser: `United States`, `Canada`, `Europe`, `Brazil`, `Latam`, `All Other Regions` ou `No Region` |
```

**Review by lfsigreja — APPROVED** [review]
menu lgtm

**Pull request merged**